website/docs/d/group_settings.md

```diff
@@ -36,116 +36,149 @@ The following arguments are supported:
+* `who_can_enter_free_form_tags` - Deprecated. This is merged into the new
+  whoCanAssistContent setting.
 
-* `who_can_hide_abuse`
+* `who_can_hide_abuse` - Deprecated. This is merged into the new
+  whoCanModerateContent setting.
 
-* `who_can_invite`
+* `who_can_invite` - Deprecated. This is merged into the new
+  whoCanModerateMembers setting.
 
-* `who_can_join`
+* `who_can_join` - Permission to join group.
 
-* `who_can_leave_group`
+* `who_can_leave_group` - Permission to leave the group.
 
-* `who_can_lock_topics`
+* `who_can_lock_topics` - Deprecated. This is merged into the new
+  whoCanModerateContent setting.
 
-* `who_can_make_topics_sticky`
+* `who_can_make_topics_sticky`  - Deprecated. This is merged into the new
+  whoCanModerateContent setting.
 
-* `who_can_mark_duplicate`
+* `who_can_mark_duplicate` - Deprecated. This is merged into the new
+  whoCanAssistContent setting.
 
-* `who_can_mark_favorite_reply_on_any_topic`
+* `who_can_mark_favorite_reply_on_any_topic` - Deprecated. This is merged into
+  the new whoCanAssistContent setting.
 
-* `who_can_mark_favorite_reply_on_own_topic`
+* `who_can_mark_favorite_reply_on_own_topic` - Deprecated. This is merged into
+  the new whoCanAssistContent setting.
 
-* `who_can_mark_no_response_needed`
+* `who_can_mark_no_response_needed` - Deprecated. This is merged into the new
+  whoCanAssistContent setting.
 
-* `who_can_moderate_content`
+* `who_can_moderate_content` - Specifies who can moderate content.
 
-* `who_can_moderate_members`
+* `who_can_moderate_members` - Specifies who can manage members.
 
-* `who_can_modify_members`
+* `who_can_modify_members` - Deprecated. This is merged into the new
+  whoCanModerateMembers setting.
 
-* `who_can_modify_tags_and_categories`
+* `who_can_modify_tags_and_categories` - Deprecated. This is merged into the
+  new whoCanAssistContent setting.
 
-* `who_can_move_topics_in`
+* `who_can_move_topics_in` - Deprecated. This is merged into the new
+  whoCanModerateContent setting.
 
-* `who_can_move_topics_out`
+* `who_can_move_topics_out` - Deprecated. This is merged into the new
+  whoCanModerateContent setting.
 
-* `who_can_post_announcements`
+* `who_can_post_announcements` - Deprecated. This is merged into the new
+  whoCanModerateContent setting.
 
-* `who_can_post_message`
+* `who_can_post_message` - Permissions to post messages.
 
-* `who_can_take_topics`
+* `who_can_take_topics` - Deprecated. This is merged into the new
+  whoCanAssistContent setting.
 
-* `who_can_unassign_topic`
+* `who_can_unassign_topic` - Deprecated. This is merged into the new
+  whoCanAssistContent setting.
 
-* `who_can_unmark_favorite_reply_on_any_topic`
+* `who_can_unmark_favorite_reply_on_any_topic` - Deprecated. This is merged into
+  the new whoCanAssistContent setting.
 
-* `who_can_view_group`
+* `who_can_view_group` - Permissions to view group messages.
 
-* `who_can_view_membership`
+* `who_can_view_membership` - Permissions to view membership.
```

Comment:
The above was copied from the URL
https://github.com/googleapis/google-api-go-client/blob/master/groupssettings/v1/groupssettings-gen.go#L146